### PR TITLE
Chore: Update sigul bridge and server hostnames

### PR DESCRIPTION
**Pull request by jwagantall**
Add sigul key information

Signed-off-by: Jessica Wagantall <jwagantall@linuxfoundation.org>

### CODE REVIEW

File: jenkins-config/global-vars-sandbox.sh [hunk 1]
```diff
@@ -8,4 +8,6 @@ RELEASE_EMAIL=collab-it+dent-release@linuxfoundation.org
 RELEASE_USERNAME=dent-release
 S3_BUCKET=dent-logs-s3-cloudfront-index
 CDN_URL=logs.dent.dev
+SIGUL_BRIDGE_IP=44.232.126.158
+SIGUL_KEY=dent-release-2021
```
Comment (MightyNerdEric) [inline jenkins-config/global-vars-sandbox.sh:12]: This should be `dent-sandbox`.

File: jenkins-config/managed-config-files/custom/sigul-config/customizedCredentialMappings.sandbox.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 ---
 customizedCredentialMappings:
-  - credentialsId: "sigul-nss-pass"
-    tokenKey: "SIGUL_NSS_PASS"
+  - credentialsId: "sigul-config-credentials"
```
Comment (MightyNerdEric) [inline jenkins-config/managed-config-files/custom/sigul-config/customizedCredentialMappings.sandbox.yaml:3]: Looks like the credentials were only created on the prod Jenkins. To do any signing tests, they'll need to be created on the sandbox too.

**Review by MightyNerdEric — CHANGES_REQUESTED** [review]

File: jenkins-config/global-vars-sandbox.sh (hunk 1 above)
Comment (jwagantall) [reply jenkins-config/global-vars-sandbox.sh:12]: oh geez.. you are right! thanks!

File: jenkins-config/managed-config-files/custom/sigul-config/customizedCredentialMappings.sandbox.yaml (hunk 2 above)
Comment (jwagantall) [reply jenkins-config/managed-config-files/custom/sigul-config/customizedCredentialMappings.sandbox.yaml:3]: let me get those going.. thanks!

**Pull request merged**